lethal-squirrels/workforce-management
# Ben edit employee

## REFERENCE

Ticket: #4 

## DESCRIPTION
Added the ability to add training programs on the employee edit page.
A. Added two view models.
B. Added view models to GET/EDIT on EmployeeController
C. Added Checkboxes to Edit Employee View
D. Edited POST/EDIT method on EmployeeController to accept the edited training programs
E. POST/EDIT queries the edited employee and then saves changes (binding happens in the method, not passed in with the Bind[] like normal). 
F. POST/EDIT calls a method which updates the EmployeeTraining join table with the selected/deleted training programs.

## STEPS TO TEST
1. Clone this branch to a local repo
2. In the package manager, run `update-database`. 
3. Test functionality
	* Run the site
	* Go to the Employee List and click "Edit" next to one of their names
	* Select and deselect some training programs and hit "Save". Go back into the edit page to see if the changes were saved.
